instrumentation/action_pack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-action_pack
 
+### v0.2.1 / 2022-10-04
+
+* (No significant changes)
```

Comment:
I thought there was a PR fixing `http.route`?

I wonder why it is missing? was it missing conventional commits?

Reply:
```suggestion
* FIXED: Ensures the correct route is add to http.route span attribute 
```